recipes-sota/config/disable-send-ip.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Disable IP reporting in Aktualizr"
```

Comment:
Should these recipes be prefixed with `aktualizr-`? Their names are not really helpful in isolation.

Reply:
I went back and forth on that. That probably makes sense.

Reply:
Done.